vagrant/scripts/wait_for_ntp.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+until ntpstat;
+do
+  systemctl restart ntpd.service
```

Comment:
Instead of restarting ntpd in a loop, we should do something like:

`ntpdate -qu 10.73.10.1`